src/app/legacy/containers/PodcastPromo/Inline.jsx

```diff
@@ -110,18 +123,47 @@ const StyledEpisodeTextWrapper = styled(PromoComponent.Card.EpisodesText)`
 `;
 
 const StyledCardLink = styled(PromoComponent.Card.Link)`
-  ${({ script }) => getLongPrimer(script)}
+  ${({ script }) => getGreatPrimer(script)}
   ${({ service }) => getSerifMedium(service)}
   display: block;
-
+  margin-top: ${GEL_SPACING_HLF_TRPL};
+  color: ${props => props.theme.palette.GREY_10};
+  &:visited {
+    .podcast-promo--visited {
+      color: ${props => props.theme.palette.GREY_6};
+    }
+  }
+  @media (max-width: ${GEL_GROUP_1_SCREEN_WIDTH_MIN}) {
+    margin-top: 0rem;
+  }
+  @media (min-width: ${GEL_GROUP_1_WIDTH_260PX}) {
+    margin-top: ${GEL_SPACING_DBL};
+  }
   @media (min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) {
     ${({ script }) => getPica(script)}
   }
 `;
 
+const StyledPodcastIconWrapper = styled.div`
+  position: absolute;
```

Comment:
Lots of different CSS here, postiion, float, display flex, align... juut checking all this is needed?

Reply:
Heya, yeah there's quite a bit of positioning 😅, basically the `position: absolute`, `float` and `transform` is to make the icon wrapper float to the bottom of the image. And then the `display: flex` and `align-items: center` is to place the podcast icon at the exact center of the wrapper.